CMakeLists.txt

```diff
@@ -420,6 +420,8 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     set (BARRIER_BUNDLE_APP_DIR ${BARRIER_BUNDLE_DIR}/Barrier.app)
     set (BARRIER_BUNDLE_BINARY_DIR ${BARRIER_BUNDLE_APP_DIR}/Contents/MacOS)
 
+#    set (CMAKE_OSX_ARCHITECTURES "arm64;x86_64") <-- disable until we get Qt working on arm64, this is the cause of the build failures
```

Comment:
I think it's best not to even add the comment. It could perhaps live in a WIP PR or something, right now it's just noise looking from code perspective.

Reply:
This has now been removed in ea66407c.